src/components/submission-common.js

```diff
@@ -321,10 +321,17 @@ export const inputsTable = props => {
               value: _.get(_.get(`${rowIndex}.source.type`, configuredInputDefinition), inputSourceLabels) || null,
               onChange: ({ value }) => {
                 const newType = _.get(value, inputSourceTypes)
-                const param = newType === 'record_lookup' ? 'record_attribute' : 'parameter_value'
-                const newSource = {
-                  type: newType,
-                  [param]: _.get(`${rowIndex}.source.${param}`, configuredInputDefinition)
+                let newSource
+                if (newType === 'none') {
+                  newSource = {
+                    type: newType
+                  }
+                } else {
+                  const param = newType === 'record_lookup' ? 'record_attribute' : 'parameter_value'
+                  newSource = {
+                    type: newType,
+                    [param]: _.get(`${rowIndex}.source.${param}`, configuredInputDefinition)
```

Comment:
Just thinking this through... If we're changing the param type, shouldn't we unset this? This looks like it would either be empty anyway (because param is changing), or store multiple values in the `configuredInputDefinition` so that it can be switched back to... but neither of those sound better than just setting this to `''` here and keeping it simple?

Reply:
Hmm good point. Yeah it is basically empty in either case so it makes sense to simply set it to `''` here. I will update it 👍